compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -161,6 +164,12 @@ struct ROCMOptions {
                      cl::desc("Enable slp vectorization in llvm opt."));
     binder.opt<bool>("iree-hip-llvm-global-isel", globalISel, cl::cat(category),
                      cl::desc("Enable global instruction selection in llvm."));
+
+    binder.opt<bool>(
+        "iree-hip-specialize-dispatches", specializeDispatches,
```

Comment:
(grumble grumble hip)
(lol thanks for being consistent here - that's more important, until we find/replace them all back to rocm!)